RELEASE.md

```diff
@@ -0,0 +1,152 @@
+# Release
+
+The Prometheus Go client library follows a release process similar to the [Prometheus server](https://github.com/prometheus/prometheus/blob/main/RELEASE.md).
+
+## Branch Management
+
+We use [Semantic Versioning](https://semver.org/).
+
+- Maintain separate `release-<major>.<minor>` branches
+- Branch protection enabled automatically for `release-*` branches
+- Bug fixes go to latest release branch, then merge to main
+- Features and changes go to main branch
+- Older release branches maintained on best-effort basis
+
+## Pre-Release Preparations
+
+1. Review main branch state:
+   - Expedite critical bug fixes
+   - Hold back risky changes
+   - Update dependencies via Dependabot PRs
+   - Check for security alerts
+
+## Cutting a Minor Release
+
+1. Create release branch:
+
+   ```bash
+   git checkout -b release-<major>.<minor> main
+   git push origin release-<major>.<minor>
+   ```
+
+2. Create feature branch:
+
+   ```bash
+   git checkout -b <yourname>/cut-<major>.<minor>.0 release-<major>.<minor>
+   ```
+
+3. Update version and documentation:
+   - Update `VERSION` file
+   - Update `CHANGELOG.md` (user-impacting changes)
+   - Order: [SECURITY], [CHANGE], [FEATURE], [ENHANCEMENT], [BUGFIX]
+   - For RCs, append `-rc.0`
+
+4. Create PR and get review
+
+5. After merge, create tags:
+
+   ```bash
+   tag="v$(< VERSION)"
+   git tag -s "${tag}" -m "${tag}"
+   git push origin "${tag}"
+   ```
+
+6. For Release Candidates:
+   - Create PR against prometheus/prometheus using RC version
+   - Create PR against kubernetes/kubernetes using RC version
+   - Make sure the CI is green for the PRs
+   - Allow 1-2 days for downstream testing
+   - Fix any issues found before final release
+   - Use `-rc.1`, `-rc.2` etc. for additional fixes
+
+7. For Final Release:
+   - Wait for CI completion
+   - Verify artifacts published
+   - Click "Publish release"
+   - For RCs, ensure "pre-release" box is checked
+
+8. Announce release:
+   - <prometheus-announce@googlegroups.com>
+   - Slack
+   - x.com/BlueSky
```

Comment:
Do you mean with our personal accounts? Or Prometheus account?

Reply:
I don't think it matters. We can use our own accounts and ask someone to repost it.

Reply:
I don't think it matters. We can use our own accounts and ask someone to repost it.